specs/protocol/isthmus/l1-attributes.md

```diff
@@ -0,0 +1,40 @@
+# L1 Block Attributes
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The L1 block attributes transaction is updated to include the EIP-1559 parameters and the extra scalars
+for the `configurableFee`.
+
+| Input arg         | Type    | Calldata bytes | Segment |
+| ----------------- | ------- | -------------- | ------- |
+| {0x098999be}      |         | 0-3            | n/a     |
+| baseFeeScalar     | uint32  | 4-7            | 1       |
+| blobBaseFeeScalar | uint32  | 8-11           |         |
+| sequenceNumber    | uint64  | 12-19          |         |
+| l1BlockTimestamp  | uint64  | 20-27          |         |
+| l1BlockNumber     | uint64  | 28-35          |         |
+| basefee           | uint256 | 36-67          | 2       |
+| blobBaseFee       | uint256 | 68-99          | 3       |
+| l1BlockHash       | bytes32 | 100-131        | 4       |
+| batcherHash       | bytes32 | 132-163        | 5       |
+| eip1559Denominator   | uint64  | 164-171        | 6       |
+| eip1559Elasticity    | uint64  | 172-179        |         |
+| configurableFeeScalar  | uint64  | 180-187        |         |
+| configurableFeeConstant    | uint64  | 188-195        |         |
```

Comment:
Do we really want or need 64 bits instead of 32 bits size for the new parameters? E.g. the `(blob)baseFeeScalar`s also worked with 32 bits (and also a decimal scaling factor, see other comment).

Reply:
Thanks for the feedback! I agree with your point about renaming to `operatorFee` and allowing for 6 decimal points of precision, but I was a little unsure about reducing the bit width of the `operatorFeeConstant` and `operatorFeeScalar`. 

I think it should be fine to decrease the `Scalar` to 32 bits, but I'm concerned that 32 bits won't be enough to represent the constant factor. For example, in this transaction [https://optimistic.etherscan.io/tx/0xa6dfc18c35bf39fa60823e9280bde18496e27e9016040f7ad9ded6797c374f05](transaction), the total transaction fee in wei requires 43 bits to represent.

If we scale the constant term by a fixed factor we could fit it in 32 bits. But I don't know how much control a user might want over this constant.